File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/dardo.py
distDardo1: float; distDardo2: float; distDardo3: float; maior: float

print("Digite as tres distancias:")
distDardo1 = float(input())
distDardo2 = float(input())
distDardo3 = float(input())

if distDardo1 > distDardo2 and distDardo1 > distDardo3:
	maior = distDardo1
elif distDardo2 > distDardo3:
	maior = distDardo2
else:
	maior = distDardo3

print(f"MAIOR DISTANCIA = {maior:.2f}")

File: nelio-alves/algoritmo-e-logica-de-programacao/java/src/exercicios/Dardo.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class Dardo {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		double distDardo1, distDardo2, distDardo3, maior;

	    System.out.println("Digite as tres distancias:");
		distDardo1 = sc.nextDouble();
		distDardo2 = sc.nextDouble();
		distDardo3 = sc.nextDouble();

		if ((distDardo1 > distDardo2) && (distDardo1 > distDardo3)) {
			maior = distDardo1;
		} else if (distDardo2 > distDardo3) {
			maior = distDardo2;
		} else {
			maior = distDardo3;
		}

	    System.out.printf("MAIOR DISTANCIA = %.2f\n", maior);

		sc.close();
	}
}
